### ISSUE TITLE
http://ci.citizensnpcs.net/ is again down 

### ISSUE THREAD

**Issue by Mykekeul35** [CitizensDev/Citizens#585]
http://ci.citizensnpcs.net/ is again down ... Where could I get the lastest dev build ?

**Comment by Fireelf15** [comment]
I get the same thing! ;( please fix it modders!

**Comment by ghost** [comment]
Same :(

**Comment by Fireelf15** [comment]
It seems like the URL changed to http://www.citizensnpcs.net/

**Comment by Mykekeul35** [comment]
@Fireelf15 
No, dev build (or experimental build) isn't availaible on this website (0ko)

**Comment by xKarolusx** [comment]
same problem:
http://i42.tinypic.com/2ms12qh.png

**Comment by Mykekeul35** [comment]
@aPunch Where could we download the lastest build ? Thx

**Comment by Fireelf15** [comment]
Try bukkit dev, it should be there. I think the -dev-dev is a bug.

**Comment by Mykekeul35** [comment]
The website (http://ci.citizensnpcs.net/) is working ! Go download the lastest build
Build 357   9 janv. 2012 00:00:34    530KB

**Comment by xKarolusx** [comment]
still same error

**Comment by Fireelf15** [comment]
Its still working, what are you talking about?

**Comment by xKarolusx** [comment]
website is working but plugin have the same error...
